packages/widgetbook_core/pubspec.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
   flutter:
     sdk: flutter
   freezed_annotation: ^2.2.0
+  equatable: ^2.0.5
```

Comment:
fine for now but since we already use freezed we should go with the manual approach if we need to override equatable operators

Reply:
Reverted dependency on equatable as mentioned in a previous comment